cgmanifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "isort", "Version": "4.2.15" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "autopep8", "Version": "1.3" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "pycodestyle", "Version": "2.3.1" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "pip",
+        "pip": { "Name": "pydevd", "Version": "1.9.2" }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "Name": "winappdbg",
+          "RepositoryUrl": "https://github.com/fabioz/winappdbg",
+          "CommitHash": "1603870dc3fa3d2984ef23b6d77e400fb0a21b99"
+        },
+        "DevelopmentDependency": false
+      }
+    },
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "Name": "bytecode",
+          "RepositoryUrl": "https://github.com/vstinner/bytecode",
+          "CommitHash": "e3e77fb690ed05ac171e15694e1c5d0e0dc34e86"
+        },
+        "DevelopmentDependency": false
+      }
+    }
+  ]
+}
```

Comment:
I'm not sure about the things for which we have just some code copied and pasted.

i.e.:

There are some files extracted from python itself for backward compatibility in:
- _pydev_imps
- _pydevd_bundle\pydevconsole_code_for_ironpython.py
- third_party\pep8\lib2to3

There are some things from IPython -- see:
- pydev_ipython\README

There's a copy of the debug adapter protocol (it's MS copyright, so, maybe nothing needs to be added).
- _pydevd_bundle\_debug_adapter\debugProtocol.json

Reply:
For these (Python and IPyhton) there should be a license entry in the https://github.com/microsoft/debugpy/blob/master/src/debugpy/ThirdPartyNotices.txt . Looks like it does have these covered. They won't be picked up automatically. 

We may have to add the DAP license.